fastlane/Fastfile

```diff
@@ -39,6 +39,36 @@ platform :android do
     attach_changelog_to_main(options)
   end
 
+  desc "Tag release version with latest if necessary"
+  lane :tag_release_with_latest_if_needed do |options|
+    release_version = options[:release_version]
+
+    current_branch = git_branch
+    sh("git", "checkout", "tags/latest")
+    latest_version = android_get_version_name(gradle_file: gradle_file_path)
+    sh("git", "checkout", current_branch)
+
+    puts "Latest version is #{latest_version}"
+
+    if Gem::Version.new(release_version) > Gem::Version.new(latest_version)
+      latest_version_commit = sh("git", "rev-list", "--tags=#{release_version}*", "--max-count=1").chomp
+      puts "Tagging #{release_version}, with commit #{latest_version_commit} with latest"
+
+      add_git_tag(
+        tag: "latest",
+        force: true,
+        grouping: "releases",
+        commit: latest_version_commit
+      )
+      push_git_tags(
+        tag: "latest",
+        force: true
+      )
+    else
+      puts "There's a more recent version. Skipping tagging."
+    end
+  end
```

Comment:
nitpick: 
perhaps we can replace this with an early return to un-indent the if block: 

```ruby
unless Gem::Version.new(release_version) > Gem::Version.new(latest_version)
  puts "There's a more recent version. Skipping tagging."
  return
end

# the rest of the code here
```

Reply:
cleaner! will update